src/index.js

```diff
@@ -1,47 +1,63 @@
 import './style.css';
-
 import { isCompleted, isDone } from './status.js';
+import {
+  add, edit, remove, removeChecked,
+} from './addremove.js';
 
 const storage = window.localStorage;
-
-const task = [
-  {
-    description: 'Adding a new item',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Going shop',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Do Exercise',
-    completed: false,
-    index: 2,
-  },
-];
+const task = [];
+const addBtn = document.getElementById('addBtn');
+const clear = document.getElementById('clear');
+const form = document.getElementById('form');
 
 function render(taskStorage) {
   taskStorage.forEach((tsk) => {
     const { description } = tsk;
     const list = document.getElementById('list');
     const item = document.createElement('li');
-    const checkBox = document.createElement('input');
-    checkBox.setAttribute('type', 'checkbox');
-    checkBox.checked = isDone(tsk);
+    const checkbox = document.createElement('input');
     const itemText = document.createElement('p');
-    const threeDot = document.createElement('i');
-    threeDot.className = 'fas fa-ellipsis-v';
+    // const threeDot = document.createElement('i');
+    const removeIcon = document.createElement('i');
+
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.id = 'checkbox';
+    checkbox.checked = isDone(task);
+    // threeDot.className = 'fas fa-ellipsis-v';
+    removeIcon.className = 'far fa-trash-alt';
```

Comment:
- [ ] Kindly remove this and all other commented code, as commented code tends to make the project seem untidy.